### PR TITLE
Update and rename content/github/setting-up-and-managing-organization…

### PR DESCRIPTION
**Pull request by kawaqamislo**
…s-and-teams/managing-access-to-your-organizations-repositories.md to المحتوى/جيثب/إنشاء وإدارة المنظمات والفرق/managing-access-to-your-organizations-repositories.md

### Check off the following:
- [ ] I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
- [ ] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [ ] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

**Review by stevenans9859 — APPROVED** [review]

**Review by stevenans9859 — APPROVED** [review]

**Comment by jace4511** [comment]
> …s-and-teams/managing-access-to-your-organizations-repositories.md to المحتوى/جيثب/إنشاء وإدارة المنظمات والفرق/managing-access-to-your-organizations-repositories.md
> 
> ### Why:
> ### What's being changed:
> ### Check off the following:
> * [ ]  I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
> * [ ]  For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
> * [ ]  For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

**Comment by jace4511** [comment]
> …s-and-teams/managing-access-to-your-organizations-repositories.md to المحتوى/جيثب/إنشاء وإدارة المنظمات والفرق/managing-access-to-your-organizations-repositories.md
> 
> ### Why:
> ### What's being changed:
> ### Check off the following:
> * [ ]  I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
> * [ ]  For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
> * [ ]  For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

**Comment by jace4511** [comment]
From a44051dee244ad9125d2496d65b0169922f29328 Mon Sep 17 00:00:00 2001
From: kawaqamislo <76270760+kawaqamislo@users.noreply.github.com>
Date: Sat, 2 Jan 2021 08:04:40 +0300
Subject: [PATCH] =?UTF-8?q?Update=20and=20rename=20content/github/setting-?=
 =?UTF-8?q?up-and-managing-organizations-and-teams/managing-access-to-your?=
 =?UTF-8?q?-organizations-repositories.md=20to=20=D8=A7=D9=84=D9=85=D8=AD?=
 =?UTF-8?q?=D8=AA=D9=88=D9=89/=D8=AC=D9=8A=D8=AB=D8=A8/=D8=A5=D9=86=D8=B4?=
 =?UTF-8?q?=D8=A7=D8=A1=20=D9=88=D8=A5=D8=AF=D8=A7=D8=B1=D8=A9=20=D8=A7?=
 =?UTF-8?q?=D9=84=D9=85=D9=86=D8=B8=D9=85=D8=A7=D8=AA=20=D9=88=D8=A7=D9=84?=
 =?UTF-8?q?=D9=81=D8=B1=D9=82/managing-access-to-your-organizations-reposi?=
 =?UTF-8?q?tories.md?=
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

---
 .../managing-access-to-your-organizations-repositories.md"      | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)
 rename content/github/setting-up-and-managing-organizations-and-teams/managing-access-to-your-organizations-repositories.md => "\330\247\331\204\331\205\330\255\330\252\331\210\331\211/\330\254\331\212\330\253\330\250/\330\245\331\206\330\264\330\247\330\241 \331\210\330\245\330\257\330\247\330\261\330\251 \330\247\331\204\331\205\331\206\330\270\331\205\330\247\330\252 \331\210\330\247\331\204\331\201\330\261\331\202/managing-access-to-your-organizations-repositories.md" (99%)

diff --git a/content/github/setting-up-and-managing-organizations-and-teams/managing-access-to-your-organizations-repositories.md "b/\330\247\331\204\331\205\330\255\330\252\331\210\331\211/\330\254\331\212\330\253\330\250/\330\245\331\206\330\264\330\247\330\241 \331\210\330\245\330\257\330\247\330\261\330\251 \330\247\331\204\331\205\331\206\330\270\331\205\330\247\330\252 \331\210\330\247\331\204\331\201\330\261\331\202/managing-access-to-your-organizations-repositories.md"
similarity index 99%
rename from content/github/setting-up-and-managing-organizations-and-teams/managing-access-to-your-organizations-repositories.md
rename to "\330\247\331\204\331\205\330\255\330\252\331\210\331\211/\330\254\331\212\330\253\330\250/\330\245\331\206\330\264\330\247\330\241 \331\210\330\245\330\257\330\247\330\261\330\251 \330\247\331\204\331\205\331\206\330\270\331\205\330\247\330\252 \331\210\330\247\331\204\331\201\330\261\331\202/managing-access-to-your-organizations-repositories.md"
index b6a9fa4057e..ac108d99520 100644
--- a/content/github/setting-up-and-managing-organizations-and-teams/managing-access-to-your-organizations-repositories.md
+++ "b/\330\247\331\204\331\205\330\255\330\252\331\210\331\211/\330\254\331\212\330\253\330\250/\330\245\331\206\330\264\330\247\330\241 \331\210\330\245\330\257\330\247\330\261\330\251 \330\247\331\204\331\205\331\206\330\270\331\205\330\247\330\252 \331\210\330\247\331\204\331\201\330\261\331\202/managing-access-to-your-organizations-repositories.md"	
@@ -1,4 +1,4 @@
----
+--
 title: Managing access to your organization's repositories
 intro: Organization owners can manage individual and team access to the organization's repositories. Team maintainers can also manage a team's repository access.
 redirect_from:

**Comment by jace4511** [comment]
From a44051dee244ad9125d2496d65b0169922f29328 Mon Sep 17 00:00:00 2001
From: kawaqamislo <76270760+kawaqamislo@users.noreply.github.com>
Date: Sat, 2 Jan 2021 08:04:40 +0300
Subject: [PATCH] =?UTF-8?q?Update=20and=20rename=20content/github/setting-?=
 =?UTF-8?q?up-and-managing-organizations-and-teams/managing-access-to-your?=
 =?UTF-8?q?-organizations-repositories.md=20to=20=D8=A7=D9=84=D9=85=D8=AD?=
 =?UTF-8?q?=D8=AA=D9=88=D9=89/=D8=AC=D9=8A=D8=AB=D8=A8/=D8=A5=D9=86=D8=B4?=
 =?UTF-8?q?=D8=A7=D8=A1=20=D9=88=D8=A5=D8=AF=D8=A7=D8=B1=D8=A9=20=D8=A7?=
 =?UTF-8?q?=D9=84=D9=85=D9=86=D8=B8=D9=85=D8=A7=D8=AA=20=D9=88=D8=A7=D9=84?=
 =?UTF-8?q?=D9=81=D8=B1=D9=82/managing-access-to-your-organizations-reposi?=
 =?UTF-8?q?tories.md?=
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

---
 .../managing-access-to-your-organizations-repositories.md"      | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)
 rename content/github/setting-up-and-managing-organizations-and-teams/managing-access-to-your-organizations-repositories.md => "\330\247\331\204\331\205\330\255\330\252\331\210\331\211/\330\254\331\212\330\253\330\250/\330\245\331\206\330\264\330\247\330\241 \331\210\330\245\330\257\330\247\330\261\330\251 \330\247\331\204\331\205\331\206\330\270\331\205\330\247\330\252 \331\210\330\247\331\204\331\201\330\261\331\202/managing-access-to-your-organizations-repositories.md" (99%)

diff --git a/content/github/setting-up-and-managing-organizations-and-teams/managing-access-to-your-organizations-repositories.md "b/\330\247\331\204\331\205\330\255\330\252\331\210\331\211/\330\254\331\212\330\253\330\250/\330\245\331\206\330\264\330\247\330\241 \331\210\330\245\330\257\330\247\330\261\330\251 \330\247\331\204\331\205\331\206\330\270\331\205\330\247\330\252 \331\210\330\247\331\204\331\201\330\261\331\202/managing-access-to-your-organizations-repositories.md"
similarity index 99%
rename from content/github/setting-up-and-managing-organizations-and-teams/managing-access-to-your-organizations-repositories.md
rename to "\330\247\331\204\331\205\330\255\330\252\331\210\331\211/\330\254\331\212\330\253\330\250/\330\245\331\206\330\264\330\247\330\241 \331\210\330\245\330\257\330\247\330\261\330\251 \330\247\331\204\331\205\331\206\330\270\331\205\330\247\330\252 \331\210\330\247\331\204\331\201\330\261\331\202/managing-access-to-your-organizations-repositories.md"
index b6a9fa4057e..ac108d99520 100644
--- a/content/github/setting-up-and-managing-organizations-and-teams/managing-access-to-your-organizations-repositories.md
+++ "b/\330\247\331\204\331\205\330\255\330\252\331\210\331\211/\330\254\331\212\330\253\330\250/\330\245\331\206\330\264\330\247\330\241 \331\210\330\245\330\257\330\247\330\261\330\251 \330\247\331\204\331\205\331\206\330\270\331\205\330\247\330\252 \331\210\330\247\331\204\331\201\330\261\331\202/managing-access-to-your-organizations-repositories.md"	
@@ -1,4 +1,4 @@
----
+--
 title: Managing access to your organization's repositories
 intro: Organization owners can manage individual and team access to the organization's repositories. Team maintainers can also manage a team's repository access.
 redirect_from:

File: المحتوى/جيثب/إنشاء وإدارة المنظمات والفرق/managing-access-to-your-organizations-repositories.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
----
```
Comment (jace4511) [inline المحتوى/جيثب/إنشاء وإدارة المنظمات والفرق/managing-access-to-your-organizations-repositories.md:1]: @stevenans9859

**Comment by jace4511** [comment]
> …s-and-teams/managing-access-to-your-organizations-repositories.md to المحتوى/جيثب/إنشاء وإدارة المنظمات والفرق/managing-access-to-your-organizations-repositories.md
> 
> ### Why:
> ### What's being changed:
> ### Check off the following:
> * [ ]  I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
> * [ ]  For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
> * [ ]  For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

**Comment by jace4511** [comment]
From a44051dee244ad9125d2496d65b0169922f29328 Mon Sep 17 00:00:00 2001
From: kawaqamislo <76270760+kawaqamislo@users.noreply.github.com>
Date: Sat, 2 Jan 2021 08:04:40 +0300
Subject: [PATCH] =?UTF-8?q?Update=20and=20rename=20content/github/setting-?=
 =?UTF-8?q?up-and-managing-organizations-and-teams/managing-access-to-your?=
 =?UTF-8?q?-organizations-repositories.md=20to=20=D8=A7=D9=84=D9=85=D8=AD?=
 =?UTF-8?q?=D8=AA=D9=88=D9=89/=D8=AC=D9=8A=D8=AB=D8=A8/=D8=A5=D9=86=D8=B4?=
 =?UTF-8?q?=D8=A7=D8=A1=20=D9=88=D8=A5=D8=AF=D8=A7=D8=B1=D8=A9=20=D8=A7?=
 =?UTF-8?q?=D9=84=D9=85=D9=86=D8=B8=D9=85=D8=A7=D8=AA=20=D9=88=D8=A7=D9=84?=
 =?UTF-8?q?=D9=81=D8=B1=D9=82/managing-access-to-your-organizations-reposi?=
 =?UTF-8?q?tories.md?=
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

---
 .../managing-access-to-your-organizations-repositories.md"      | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)
 rename content/github/setting-up-and-managing-organizations-and-teams/managing-access-to-your-organizations-repositories.md => "\330\247\331\204\331\205\330\255\330\252\331\210\331\211/\330\254\331\212\330\253\330\250/\330\245\331\206\330\264\330\247\330\241 \331\210\330\245\330\257\330\247\330\261\330\251 \330\247\331\204\331\205\331\206\330\270\331\205\330\247\330\252 \331\210\330\247\331\204\331\201\330\261\331\202/managing-access-to-your-organizations-repositories.md" (99%)

diff --git a/content/github/setting-up-and-managing-organizations-and-teams/managing-access-to-your-organizations-repositories.md "b/\330\247\331\204\331\205\330\255\330\252\331\210\331\211/\330\254\331\212\330\253\330\250/\330\245\331\206\330\264\330\247\330\241 \331\210\330\245\330\257\330\247\330\261\330\251 \330\247\331\204\331\205\331\206\330\270\331\205\330\247\330\252 \331\210\330\247\331\204\331\201\330\261\331\202/managing-access-to-your-organizations-repositories.md"
similarity index 99%
rename from content/github/setting-up-and-managing-organizations-and-teams/managing-access-to-your-organizations-repositories.md
rename to "\330\247\331\204\331\205\330\255\330\252\331\210\331\211/\330\254\331\212\330\253\330\250/\330\245\331\206\330\264\330\247\330\241 \331\210\330\245\330\257\330\247\330\261\330\251 \330\247\331\204\331\205\331\206\330\270\331\205\330\247\330\252 \331\210\330\247\331\204\331\201\330\261\331\202/managing-access-to-your-organizations-repositories.md"
index b6a9fa4057e..ac108d99520 100644
--- a/content/github/setting-up-and-managing-organizations-and-teams/managing-access-to-your-organizations-repositories.md
+++ "b/\330\247\331\204\331\205\330\255\330\252\331\210\331\211/\330\254\331\212\330\253\330\250/\330\245\331\206\330\264\330\247\330\241 \331\210\330\245\330\257\330\247\330\261\330\251 \330\247\331\204\331\205\331\206\330\270\331\205\330\247\330\252 \331\210\330\247\331\204\331\201\330\261\331\202/managing-access-to-your-organizations-repositories.md"	
@@ -1,4 +1,4 @@
----
+--
 title: Managing access to your organization's repositories
 intro: Organization owners can manage individual and team access to the organization's repositories. Team maintainers can also manage a team's repository access.
 redirect_from:

**Comment by jace4511** [comment]
…s-and-teams/managing-access-to-your-organizations-repositories.md to المحتوى/جيثب/إنشاء وإدارة المنظمات والفرق/managing-access-to-your-organizations-repositories.md

**Comment by jace4511** [comment]
…s-and-teams/managing-access-to-your-organizations-repositories.md to المحتوى/جيثب/إنشاء وإدارة المنظمات والفرق/managing-access-to-your-organizations-repositories.md

**Comment by jace4511** [comment]
…s-and-teams/managing-access-to-your-organizations-repositories.md to المحتوى/جيثب/إنشاء وإدارة المنظمات والفرق/managing-access-to-your-organizations-repositories.md

**Comment by jace4511** [comment]
…s-and-teams/managing-access-to-your-organizations-repositories.md to المحتوى/جيثب/إنشاء وإدارة المنظمات والفرق/managing-access-to-your-organizations-repositories.md

**Comment by jace4511** [comment]
…s-and-teams/managing-access-to-your-organizations-repositories.md to المحتوى/جيثب/إنشاء وإدارة المنظمات والفرق/managing-access-to-your-organizations-repositories.md

**Comment by jace4511** [comment]
…s-and-teams/managing-access-to-your-organizations-repositories.md to المحتوى/جيثب/إنشاء وإدارة المنظمات والفرق/managing-access-to-your-organizations-repositories.md

**Comment by jace4511** [comment]
> …s-and-teams/managing-access-to-your-organizations-repositories.md to المحتوى/جيثب/إنشاء وإدارة المنظمات والفرق/managing-access-to-your-organizations-repositories.md
> 
> ### Why:
> ### What's being changed:
> ### Check off the following:
> * [ ]  I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
> * [ ]  For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
> * [ ]  For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

**Comment by jace4511** [comment]
> …s-and-teams/managing-access-to-your-organizations-repositories.md to المحتوى/جيثب/إنشاء وإدارة المنظمات والفرق/managing-access-to-your-organizations-repositories.md
> 
> ### Why:
> ### What's being changed:
> ### Check off the following:
> * [ ]  I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
> * [ ]  For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
> * [ ]  For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

**Comment by jace4511** [comment]
…s-and-teams/managing-access-to-your-organizations-repositories.md to المحتوى/جيثب/إنشاء وإدارة المنظمات والفرق/managing-access-to-your-organizations-repositories.md

**Review by jace4511 — COMMENTED** [review]
-----BEGIN PGP MESSAGE-----

hQIMA7TlqkAVGpiPAQ//ZDj6wq2bBibXM5CRbCk84tgK4B8ezfiAM6ev61/aA1cO
fl6IP+E1cPwg1hqU5fEhnsW8dyUBr2UIwL7apcjiW+wS572U9ojwlNwrScmSDvCj
i/ugi9iN0bC51batV3tltHl9tbdn4YmEJWEfUy24Qn8OXEGztNbSLqtb9gemm+Ba
qB5cml5q57jESH3mXtdUKOvQz9p3TorMW/U5/4WiuC2h9H6x4oayKuGfJaFXp8NP
OO3z7m2x/+ozi8gsCuwoEMs82vphWP6XXCXOpZPTaukqgm31xvYMHi13FgXmUuIP
65kU+DXIVrMVTCQPBDb/fETB6zK1vVRl+h1LheCqSkSAE9YdtHyMfF3veyr93g62
oYFSugvu2UbIQw4odk+/oR5zx8eo+q2BiLt2V3hGVhGnKtWjY0gE0wweOd7Orrhe
8r4QNbDwswKWmXMuMvACgzmx1H+3QLnxzFhCpEZNEujoTl+/M38ZkWjejTTJr+ir
XwEagzcLtEC4QGamdacL2C+CtHSP/o6uhZymNTEVd5WXyQhfwC/vPszPk2uZJe/w
1RcozKN5yNUDnUAH/hU3f9mvvu/lTEpS1PO4e1yTH35OKX2EZBNxzVpyTeQ09NLX
+bwO1/svEu6sRBZ4kWOWnJoG58Ak/TzkvKuIsiEWXzOGoqBHxuQxe8R+EwSnL2vS
wK0BKsRyMKfsbRMCiXmGXYyoICnazad4zU7HfU7GFOzNJWSRwkplEV2eIn6laNXJ
Xk0+TkGAxne9gGbNwX/lN7mrB1JoBngXljdc+cphHjC5M/hcqC6pWvyxdIfcnpOc
bGut69Is2iRjZnPGRt4mqqqyXHVfOrllM31UI6RLbHPPOn9T7OHRisLxhwgzZvZ3
omV1iKmNIkxISQ/fongFcLQ8vR5cXxEk7KM7Sw0fy9uFk2eIRtI5qbIGWUk1ObUE
bUS42gxKVpgG2+MWAyoTmwHhA6j+rpig00rhDJl1lWm9ip9dDz7fD/hl+kMMxYq3
qBePHLdocUkhVtTbHFmfbhkSvB4Vh7ZjKwYVJHOgMQ/jU3NnZ4VXVJ1zyqxY/9CH
hcm6aL97bon1kQDy6yepBtMzGJ7jQGLHDeB5YtKBkv6d12zlmNUvzrvsAb2oRkHI
Dkn43kUm3lRL4YPWH0BHx8T78crtBtIdAPI0o0dbHQ==
=pRAi
-----END PGP MESSAGE-----

**Review by jace4511 — COMMENTED** [review]
Duplicate of #

**Review by stevenans9859 — COMMENTED** [review]
@Stevenans985900

**Comment by jace4511** [comment]
> …s-and-teams/managing-access-to-your-organizations-repositories.md to المحتوى/جيثب/إنشاء وإدارة المنظمات والفرق/managing-access-to-your-organizations-repositories.md
> 
> ### Why:
> ### What's being changed:
> ### Check off the following:
> * [ ]  I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
> * [ ]  For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
> * [ ]  For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

**Comment by jace4511** [comment]
> …s-and-teams/managing-access-to-your-organizations-repositories.md to المحتوى/جيثب/إنشاء وإدارة المنظمات والفرق/managing-access-to-your-organizations-repositories.md
> 
> ### Why:
> ### What's being changed:
> ### Check off the following:
> * [ ]  I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
> * [ ]  For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
> * [ ]  For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

**Pull request closed without merging**